app/lib/actions/pulp3/repository/repair.rb

```diff
@@ -23,6 +23,10 @@ def run_progress_weight
         def presenter
           Presenters::RepairPresenter.new(self)
         end
+
+        def rescue_strategy_for_self
+          Dynflow::Action::Rescue::Skip
+        end
```

Comment:
this is not strictly required, but i noticed it when i reproduced that the task failed and caused a paused task instead of ending